### PR TITLE
Update sum.mdx docs

### PR DESCRIPTION
**Pull request by AgustinRamiroDiaz**
Explain edge case with empty list

Fixes #

## Target Release

1.5.x

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

**Comment by crw** [comment]
Thanks for this submission! I have notified the docs team for triage.

File: website/docs/language/functions/sum.mdx [hunk 1]
```diff
@@ -9,6 +9,8 @@ description: |-
 
 `sum` takes a list or set of numbers and returns the sum of those numbers.
 
+`sum` will fail if given an empty list or set.
```
Comment (trujillo-adam) [inline website/docs/language/functions/sum.mdx:12]:
```suggestion
`sum` fails when given an empty list or set.
```
Use future tense per our style guide.

**Review by trujillo-adam — COMMENTED** [review]
Thanks for adding this! Just a small suggestion for consistency with our writing style.

**Review by trujillo-adam — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**